ocean_lib/web3_internal/wallet.py

```diff
@@ -123,20 +123,35 @@ def sign_tx(
             )
         else:
             nonce = Wallet._get_nonce(self.web3, account.address)
+            if not gas_price:
+                try:
+                    history = self.web3.eth.fee_history(
+                        block_count=1, newest_block="latest"
+                    )
+                    tx["type"] = "0x2"
+                    tx["maxPriorityFeePerGas"] = self.web3.eth.max_priority_fee
+                    tx["maxFeePerGas"] = (
+                        self.web3.eth.max_priority_fee + history["baseFeePerGas"][0] * 2
+                    )
+                    tx["gas"] = GAS_LIMIT_DEFAULT
+                except ValueError as e:
+                    assert (
+                        e.args[0]["message"] == "Method eth_feeHistory not supported."
+                    )
+
+                    from ocean_lib.web3_internal.contract_base import ContractBase
```

Comment:
why is thihs import here and not at the top of the file?

Reply:
It will generate a circular import error if I put at the top of the file.